server/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -496,7 +552,7 @@ private void executeSearch(SearchTask task, SearchTimeProvider timeProvider, Sea
             searchRequest.searchType(QUERY_THEN_FETCH);
         }
         if (searchRequest.allowPartialSearchResults() == null) {
-           // No user preference defined in search request - apply cluster service default
+            // No user preference defined in search request - apply cluster service default
```

Comment:
We have already set this on line 500, do we need this again here?

Reply:
Addressed in https://github.com/elastic/elasticsearch/commit/0bc08e15223251b4ddb2d099f3a17222bf991a9f.